doc/manual/parallel-computing.rst

```diff
@@ -55,31 +42,134 @@ equal the number of CPU cores on the machine.
 
     $ ./julia -p 2
 
+You can check the number of available processes with :func:`nprocs`.
+
+::
+
+    julia> nprocs()
+    3
+
+The processes used by default for parallel operations are referred to as "workers".
+When there is only one process, process 1 is considered a worker. Otherwise, workers are
+considered to be all processes other than process 1, which is called the "master"
+process. In the above example, there are a total of 3 processes (2 workers, 1 master).
+To check the number of worker processes, you can use :func:`nworkers`.
+
+::
+
+    julia> nworkers()
+    2
+
+Each process has a unique integer id. You can use the :func:`procs` command to
+see them.
+
+::
+
+    julia> procs()
+    3-element Array{Int64,1}:
+     1
+     2
+     3
+
+You can remove processes using :func:`rmprocs`
+
+::
+
+    julia> rmprocs(2) # removes process 2
+    :ok
+
+    julia> procs()
+    2-element Array{Int64,1}:
+     1
+     3
+
+You can add processes using :func:`addprocs` (an alternative to using
```

Comment:
Maybe just link to: http://docs.julialang.org/en/latest/manual/parallel-computing/#specifying-network-topology-experimental?